### DISCUSSION TITLE
QUploader computedProps

### DISCUSSION THREAD

**Discussion by anazzani** [quasarframework/quasar#15887] [General - Components / Directives / etc]
Hi all.

I'm trying to access the q-uploader computedProps (namely files and/or queuedFiles) in my SFC using Composition API but I always get an empty Array, even after selecting one or more files: am I misunderstanding their purpose?

https://stackblitz.com/edit/quasarframework-tkmagz

TIA.

Alessandro

**Comment by yusufkandemir** [comment]
See #14967
